### ISSUE TITLE
semidominant inheritance

### ISSUE THREAD

**Issue by maglott** [obophenotype/human-phenotype-ontology#7172]
Would it be possible to change the preferred label for https://hpo.jax.org/app/browse/term/HP:0032113 to 
Semidominant inheritance?  

This seems to be the only child of Mode of inheritance that uses the word 'mode', and ClinGen asked ClinVar to validate 'semidominant inheritance'

Thanks

**Comment by pnrobinson** [comment]
good suggestion, thanks, I have changed the primary label and made the original label a synonym
